internal/provider/waypoint/resource_waypoint_add_on_definition.go

```diff
@@ -109,9 +109,17 @@ func (r *AddOnDefinitionResource) Schema(ctx context.Context, req resource.Schem
 				Description: "The markdown template for the Add-on Definition README (markdown format supported).",
 				Optional:    true,
 			},
+			"terraform_project_id": schema.StringAttribute{
+				Required:    true,
+				Description: "The ID of the Terraform Cloud Project to create workspaces in",
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
 			"terraform_cloud_workspace_details": &schema.SingleNestedAttribute{
-				Optional: true,
-				Computed: true,
+				DeprecationMessage: "terraform_cloud_workspace_details is deprecated, use terraform_project_id instead",
```

Comment:
That is cool that you are pointing to what to use, but can you do the extra mile for our user by pointing how to get that information?

Reply:
We don't do this already for the existing param so I was following what has already been merged: https://registry.terraform.io/providers/hashicorp/hcp/latest/docs/resources/waypoint_tfc_config#optional

I'm fine adding more docs. What is expected here though on this deprecation message? Are you asking about how to get `terraform_project_id` value when you say "do the extra mile for our user by pointing how to get that information?"? Wouldn't that be on that fields docs, and not docs about `terraform_cloud_workspace_details` being deprecated?

Reply:
Sure.

Reply:
[52d7f51](https://github.com/hashicorp/terraform-provider-hcp/pull/1052/commits/52d7f512dd4f62f73cc335726b105b97a925a2ce)